goveralls.go

```diff
@@ -248,8 +251,14 @@ func process() error {
 	//
 	// Initialize Job
 	//
+
+	// flags are never nil, so no nil check needed
 	var jobId string
-	if travisJobId := os.Getenv("TRAVIS_JOB_ID"); travisJobId != "" {
+	if *customJobId != "" {
+		jobId = *customJobId
+	} else if serviceJobId := os.Getenv("COVERALLS_SERVICE_JOB_ID"); serviceJobId != "" {
```

Comment:
renamed to `COVERALLS_SERVICE_JOB_ID`
it is compatible with node-coveralls https://github.com/nickmerwin/node-coveralls#usage